CodeTrans/tests/test_codetrans_on_xeon.sh

```diff
@@ -19,10 +19,10 @@ function build_docker_images() {
     git clone https://github.com/opea-project/GenAIComps.git
 
     echo "Build all the images with --no-cache, check docker_image_build.log for details..."
-    service_list="codetrans codetrans-ui llm-tgi"
+    service_list="codetrans codetrans-ui llm-tgi nginx"
     docker compose -f docker_build_compose.yaml build ${service_list} --no-cache > ${LOG_PATH}/docker_image_build.log
 
-    docker pull ghcr.io/huggingface/text-generation-inference:1.4
+    docker pull ghcr.io/huggingface/text-generation-inference:latest-intel-cpu
```

Comment:
ghcr.io/huggingface/text-generation-inference:sha-e4201f4-intel-cpu should be currently the right target. But since the image name in compose.yaml is sha-e4201f4-intel-cpu, the CI just passed. Maybe here we just follow the same tag `sha-e4201f4-intel-cpu` here?

Reply:
Is the tag `sha-e4201f4-intel-cpu` stable in v1.0? Will not merge this pr in v0.9, need to make sure using the stable version of docker image.

Reply:
ok, let's recheck this after v0.9 release